packages/react-native-sdk/__tests__/mocks/call.ts

```diff
@@ -17,5 +18,9 @@ export const mockCall = (
     OwnCapability.SEND_AUDIO,
     OwnCapability.SEND_VIDEO,
   ]);
+  if (ownCapabilities) {
+    call.permissionsContext.setPermissions([...ownCapabilities]);
+    call.state.setOwnCapabilities(ownCapabilities);
```

Comment:
Rather no if else logics in mocks, it can get unneeded complex, please set it to a value if provided or a default value if not ;)

Reply:
Done